lib/screen/club_detail/club_detail_screen.dart

```diff
@@ -20,35 +24,60 @@ class ClubDetailScreen extends StatefulWidget {
 
 @visibleForTesting
 class ClubDetailScreenState extends State<ClubDetailScreen> with BackNavigatorMixin, ErrorNavigatorMixin implements ClubDetailViewNavigator {
-
   @override
   Widget build(BuildContext context) {
     return ProviderWidget<ClubDetailViewModel>(
-      create: () => GetIt.I()..init(this),
+      create: () => GetIt.I()..init(this, widget.clubId),
       consumerWithThemeAndLocalization: (context, viewModel, child, theme, localization) {
         return Scaffold(
           backgroundColor: theme.colorsTheme.backgroundDark,
           body: viewModel.club == null
               ? const Center(
                   child: CircularProgressIndicator(),
                 )
-              : Center(
-                  child: Stack(
+              : ListView(children: [
+                  Stack(
                     children: [
                       ClubBanner(imgUrl: viewModel.club!.coverPhoto!),
                       Padding(
                         padding: const EdgeInsets.fromLTRB(ThemeDimens.padding56, 270, ThemeDimens.padding56, 0),
                         child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
                           children: [
                             ClubHeader(
                               club: viewModel.club!,
                             ),
+                            const SizedBox(
+                              height: ThemeDimens.padding16,
+                            ),
+                            viewModel.members == null
```

Comment:
you are writing the same thing twice. It is better to create a new widget for that